todo_app/app.py

```diff
@@ -1,18 +1,50 @@
 from flask import Flask, redirect, render_template, request
-from todo_app.data.session_items import get_items, add_item
+from itertools import chain
 from todo_app.flask_config import Config
+from todo_app.Item import Item
+import os
+import requests
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
+board_id = os.getenv('BOARD_ID')
+api_key = os.getenv('API_KEY')
+token = os.getenv('TOKEN')
+base_url = f'https://api.trello.com/1'
+base_query_params = {
+    'key': api_key,
+    'token': token,
+    'cards': 'open',
+}
+list_url = f'{base_url}/boards/{board_id}/lists'
+list_query_params = query_params = dict(base_query_params, **{'cards': 'open'})
 
 @app.route('/')
 def index():
-    to_do_items = get_items()
-    return render_template('index.html', to_do_items = to_do_items)
+    lists = requests.get(list_url, list_query_params).json()
+    print(lists)
+    nested_cards = [[Item.from_trello_card(card, list) for card in list['cards']] for list in lists]
+    cards = list(chain.from_iterable(nested_cards))
+    return render_template('index.html', cards = cards)
 
 
 @app.route('/submit', methods=["POST"])
 def submit_form():
-    add_item(request.form.get('title'))
-    return redirect('/')
\ No newline at end of file
+    url = f'{base_url}/cards'
```

Comment:
Not necessary to change this now, but it would've been cleaner to separate the Trello logic into a separate service, e.g. `trello_items.py`, similar to how you used `session_items.py`. Also, you don't need `session_items.py` anymore